src/components/encoding-pane/encoding-shelf.tsx

```diff
@@ -5,6 +5,7 @@ import * as CSSModules from 'react-css-modules';
 import {ConnectDropTarget, DropTarget, DropTargetCollector, DropTargetSpec} from 'react-dnd';
 import * as TetherComponent from 'react-tether';
 import {contains} from 'vega-lite/build/src/util';
+import {isValueDef} from '../../../node_modules/vega-lite/build/src/fielddef';
```

Comment:
Remove `../../../node_modules/`.  Please also check if you have accidentally included `node_modules` like this in other places too.  :)